docs/sqlite-integration.md

```diff
@@ -0,0 +1,161 @@
+# SQLite WASM Integration Documentation
+
+This document describes the SQLite WASM integration in the Zebra LLM Conversation Cache extension, including how to interact with the database and use the available functions.
+
+## Overview
+
+The extension uses SQLite WASM with Origin-Private FileSystem (OPFS) support to store conversation data persistently. The implementation handles both service worker and regular contexts, providing a unified API for database operations.
```

Comment:
>  The extension uses SQLite WASM with Origin-Private FileSystem (OPFS) support

I don't think this is currently true. Is it?

I see this message in the service worker logs:
> OPFS is not available